.circleci/images/ubuntu_1804/Dockerfile

```diff
@@ -0,0 +1,34 @@
+FROM phusion/baseimage:0.11
+
+MAINTAINER Mohit Agnihotri "mohitkumar39@gmail.com"
+
+WORKDIR /home/circleci
+ENV HOME /home/circleci
+
+RUN apt-get -qq update &&  apt-get -qqy upgrade
+
+RUN apt-get install -y --no-install-recommends git cmake ninja-build gperf \
+         ccache doxygen dfu-util device-tree-compiler \
+         python3-ply python3-pip python3-setuptools python3-wheel xz-utils file \
+         make gcc-multilib autoconf automake libtool librsvg2-bin git python3-dev \
+         libgtest-dev g++ make cmake wget build-essential clang clang-tidy ninja-build
+
+RUN	cd /usr/src/gtest && \
```

Comment:
It seems that you used some commands from .circleci/images/primary/Dockerfile. I found out that it uses both spaces and tabs for indentation. I suggest that you should use only spaces or only tabs for indentation not using them both.

Reply:
I second this.